File: app/src/main/java/com/mifos/apache/fineract/data/models/payment/CostComponent.java
package com.mifos.apache.fineract.data.models.payment;

/**
 * @author Rajan Maurya
 *         On 13/07/17.
 */

public class CostComponent {

    private String chargeIdentifier;
    private Double amount;

    public String getChargeIdentifier() {
        return chargeIdentifier;
    }

    public void setChargeIdentifier(String chargeIdentifier) {
        this.chargeIdentifier = chargeIdentifier;
    }

    public Double getAmount() {
        return amount;
    }

    public void setAmount(Double amount) {
        this.amount = amount;
    }
}

File: app/src/main/java/org/apache/fineract/data/models/payment/CostComponent.kt
package org.apache.fineract.data.models.payment

import com.google.gson.annotations.SerializedName

/**
 * @author Rajan Maurya
 * On 13/07/17.
 */

data class CostComponent (
    @SerializedName("chargeIdentifier") var chargeIdentifier: String? = null,
    @SerializedName("amount") var amount: Double? = null
)
